examples/burial-form/MilitaryServiceHistory.jsx

```diff
@@ -1,18 +1,71 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
 import React from 'react';
 import {
+  ArrayField,
   DateField,
   Page,
   TextField,
   SelectField,
+  DebuggerView
 } from '@department-of-veterans-affairs/va-forms-system-core';
 import { useFormikContext } from 'formik';
 import { isBeforeDate } from './utils';
-import { VaButton } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const ARRAY_FIELD_PROPS = {
```

Comment:
Should this be moved to a types file?

Reply:
@bjmccotter7192 this is the customization the user is doing when they customize the array field. Technically it's not a type because its JSX

Reply:
@bjmccotter7192 I added some more docs related to this. Let me know if theres still any questions about how to use it.